lib/bolt/transport/ssh.rb

```diff
@@ -102,6 +102,7 @@ def initialize
 
         require 'net/ssh'
         require 'net/scp'
```

Comment:
```suggestion
        begin
          require 'net/ssh/krb'
        rescue LoadError
          logger.debug("Authentication method 'gssapi-with-mic' (Kerberos) is not available.")
        end
```

Reply:
I opted not to have a more specific message here, because we'd be assuming the cause of the load error. There may be other issues than just running on Windows that cause a loaderror.

Reply:
Would committing this suggestion as is delete `require 'net/scp'`?